test/testcafe/test-snippet-beta.js

```diff
@@ -1,5 +1,5 @@
 !function(name,path,ctx){
-  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.parentNode.insertBefore(s,h)}}
+  var latest,prev=name!=='Keen'&&window.Keen?window.Keen:false;ctx[name]=ctx[name]||{ready:function(fn){var h=document.getElementsByTagName('head')[0],s=document.createElement('script'),w=window,loaded;s.onload=s.onreadystatechange=function(){if((s.readyState&&!(/^c|loade/.test(s.readyState)))||loaded){return}s.onload=s.onreadystatechange=null;loaded=1;latest=w.Keen;if(prev){w.Keen=prev}else{try{delete w.Keen}catch(e){w.Keen=void 0}}ctx[name]=latest;ctx[name].ready(fn)};s.async=1;s.src=path;h.parentNode.appendChild(s)}}
```

Comment:
@adewinter this looks like it's adding the script below `<body>`. Have you tried this out?

`headEl.parent.appendChild` should be more like scriptEl.parent.appendChild`

Reply:
Ah, looks like you already removed the parent mode logic from the other snippets. Just need this one too

Reply:
Ah good catch! Yeah, def want just `headEl.appendChild` since there may or may not be any `<script>` tags to begin with we can't rely on `scriptEl.parent !== undefined`.  Will fix